staging/src/k8s.io/api/flowcontrol/v1beta1/types.go

```diff
@@ -505,6 +515,43 @@ type LimitedPriorityLevelConfiguration struct {
 	BorrowingLimitPercent *int32 `json:"borrowingLimitPercent,omitempty" protobuf:"varint,4,opt,name=borrowingLimitPercent"`
 }
 
+// ExemptPriorityLevelConfiguration describes the configurable aspects
+// of the handling of exempt requests.
+// In the mandatory exempt configuration object the values in the fields
+// here can be modified by authorized users, unlike the rest of the `spec`.
+type ExemptPriorityLevelConfiguration struct {
+	// `nominalConcurrencyShares` (NCS) contributes to the computation of the
+	// NominalConcurrencyLimit (NominalCL) of this level.
+	// This is the number of execution seats nominally reserved for this priority level.
+	// This DOES NOT limit the dispatching from this priority level
+	// but affects the other priority levels through the borrowing mechanism.
+	// The server's concurrency limit (ServerCL) is divided among all the
+	// priority levels in proportion to their NCS values:
+	//
+	// NominalCL(i)  = ceil( ServerCL * NCS(i) / sum_ncs )
+	// sum_ncs = sum[priority level k] NCS(k)
+	//
+	// Bigger numbers mean a larger nominal concurrency limit,
+	// at the expense of every other priority level.
+	// This field has a default value of zero.
```

Comment:
I don't see this actually defaulted.  It's ok to indicate: "unspecified is equivalent to zero".

Reply:
I am not sure I understand what you mean by that field not being "actually defaulted".  When a field is left unspecified, it normally gets its default value.  That is what is going on here too.

Reply:
This is true for non-pointers, but once we switch to pointer, the default will effectively be "nil". And then we should probably add a defaulting function, like here:
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/flowcontrol/v1beta3/defaults.go#L47

Reply:
Done in the force-push to 90d57e6a7e7